dnf-behave-tests/features/updateinfo.feature

```diff
@@ -312,3 +312,32 @@ Examples:
     # yum compatibility
     | list bugzillas      |
     | list bzs            |
+
+
+@bz1728004
+Scenario: updateinfo show <advisory> of the running kernel after a kernel update
+   When I execute dnf with args "install kernel"
+   Then the exit code is 0
+    And Transaction is following
+        | Action        | Package                                  |
+        | install       | kernel-core-0:4.18.16-300.fc29.x86_64    |
+        | install       | kernel-modules-0:4.18.16-300.fc29.x86_64 |
+        | install       | kernel-0:4.18.16-300.fc29.x86_64         |
+  Given I use repository "dnf-ci-fedora-updates"
+    And I execute dnf with args "updateinfo list kernel"
+   Then the exit code is 0
+    And stdout matches line by line
```

Comment:
I think you can use the `stdout is` step here, even though the difference is mall in this case. One advantage of `stdout is` is it prints a colored diff in case of mismatch, this one doesn't (even though it could and should, just that nobody implemented it yet :slightly_smiling_face: )